ansible/library/extract_log.py

```diff
@@ -149,21 +145,41 @@ def comparator(l, r):
 
 
 def filename_comparator(l, r):
+    """Compares log filenames (older > newer), assumes file with greater number is
```

Comment:
Can you please explain, what is the sort order here
 From older to newer or otherwise?

Reply:
The sort order depends whether reversed param is True or False in sort function. By default, since older log file is considered greater then newer, the sort order will be from newer to older:
`>>> f = ['syslog', 'syslog.1', 'syslog.3.gz', 'syslog.2.gz']`
`>>> sorted(f, cmp=filename_comparator)`
`['syslog', 'syslog.1', 'syslog.2.gz', 'syslog.3.gz']`
This order was chosen for convenience, since we need to start looking from the most recent log file (from the begging of the list)